frontend/app/routes/ceqr-intro-page.js

```diff
@@ -0,0 +1,4 @@
+import Route from '@ember/routing/route';
+
+export default class CeqrIntroPageRoute extends Route {
+}
```

Comment:
`ember g route` generates this for you, but you don't actually need it if you're not using it.